themes/bootstrap3/templates/combined/stack-grid.phtml

```diff
@@ -0,0 +1,37 @@
+<div class="combined-search-container grid"
+  style="grid-template-columns: repeat(<?=$this->escapeHtmlAttr($columns)?>, 1fr)"
+>
+  <?php $columnedParams = []; ?>
+  <?php $columnIndex = 0; ?>
```

Comment:
This value is never incremented. Could that mean that `supportsCartOptions` is not being correctly applied below?

Reply:
Yes you are probably right; I added back the incrementor to match stack-distributed.  The variable name is misleading, $columnIndex is not about columns, it's about the individual search result modules.

So possibly fixed now, but I'm reading through code and wiki and not seeing how to actually enable the cart to confirm it.  Still looking

Reply:
> Still looking

Ok I found `showBulkOptions` and enabled it.  I see checkboxes next to the modules that are supposed to have them, so I think it's ok now.

Reply:
Thanks for tracking that down. Yes, it looks good now!